src/main/java/com/hartwig/platinum/pdl/ConvertForBiopsies.java

```diff
@@ -0,0 +1,50 @@
+package com.hartwig.platinum.pdl;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import com.hartwig.api.RunApi;
+import com.hartwig.api.SampleApi;
+import com.hartwig.api.SetApi;
+import com.hartwig.api.model.DbStatus;
+import com.hartwig.api.model.Ini;
+import com.hartwig.api.model.Run;
+import com.hartwig.api.model.SampleType;
+import com.hartwig.pdl.ImmutablePipelineInput;
+import com.hartwig.pdl.PipelineInput;
+import com.hartwig.pdl.generator.PdlGenerator;
+import com.hartwig.platinum.config.PlatinumConfiguration;
+
+public class ConvertForBiopsies implements PDLConversion {
+
+    private final RunApi runApi;
+    private final SampleApi sampleApi;
+    private final SetApi setApi;
+    private final PdlGenerator pdlGenerator;
+
+    public ConvertForBiopsies(final RunApi runApi, final SampleApi sampleApi, final SetApi setApi,
+            final PdlGenerator pdlGenerator) {
+        this.runApi = runApi;
+        this.sampleApi = sampleApi;
+        this.setApi = setApi;
+        this.pdlGenerator = pdlGenerator;
+    }
+
+    @Override
+    public List<PipelineInput> apply(final PlatinumConfiguration configuration) {
+        return configuration.sampleIds()
```

Comment:
A little explanation, this is a bit of a workaround to the fact that PDLGenerator works on run id. Here we find the most recent somatic run for the biopsy and generate the PDL based off that. Then we clear the operational references to make the run ephemeral. 

You could argue that we should make it such that the generator itself can work directly off the biopsy (has a reference to the sample and therefore set), and I wouldn't argue. 

But... this code should be reasonably robust and its quite simple.

Reply:
Yeah I like it. Maybe change to `parallelStream` for performance ;) (actually here it kind of makes sense, since this method is super IO heavy haha)

Reply:
Yes!